### PR TITLE
P1 navigation model

### PR DESCRIPTION
**Pull request by Ramandeep9898**

### CODE REVIEW

File: style/elements/buttons.css [hunk 1]
```diff
@@ -60,9 +63,11 @@
     font-size: var(--cta-btn-font-size);
     font-weight: 500;
     margin-bottom: 40px;
```
Comment (dhimanaishwarya) [inline style/elements/buttons.css:65]: for values above 10px you can use rem

File: style/elements/doc-centre-grid.css [hunk 2]
```diff
@@ -2,10 +2,10 @@
     justify-content: space-between;
     flex-flow: row wrap;
     align-items: stretch;
-    column-gap: 8px;
+    /* column-gap: 8px; */
```
Comment (dhimanaishwarya) [inline style/elements/doc-centre-grid.css:5]: comments can be removed , rest looks good

**Review by anshulraheja — APPROVED** [review]
if possible next time share the demo link also
Rest everything is fine, you can merge.

File: style/components/model.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    font-weight: 500;
+    cursor: pointer;
+}
+.model:target{
+    opacity: 1;
+    visibility: visible;
+
+}
+
+.model:target > .model-body{
```
Comment (PuneetP16) [inline style/components/model.css:45]: The uneven spacing could be avoided been clases. Use some auto-formatter like prettier or es-lint to fix this.

File: style/components/model.css [hunk 4]
```diff
@@ -0,0 +1,51 @@
+.model{
+    height: 100vh;
+    width: 100%;
+    background-color: rgba(0, 0, 0,0.8);
```
Comment (PuneetP16) [inline style/components/model.css:4]: Could use variables instead of direct values.

**Review by PuneetP16 — APPROVED** [review]
Good to merge. There is no preview link for reviewing your UI. Maybe you could add images of component  for their reviewal.

**Pull request merged**